mini-components/Buttons/discordButton.tsx

```diff
@@ -0,0 +1,17 @@
+"use client";
+const DiscordButton = () => {
+  const handleDiscordClick = () => {
+    window.open("https://discord.gg/B99yGVK5K8", "_blank");
```

Comment:
Is this link functional all the time, does it have an expiry date?

Reply:
it works

Reply:
ok